README.md

```diff
@@ -518,6 +517,10 @@ These are values that represent the current state of the downshift component.
 | `isOpen`           | `boolean`         | the menu open state                            |
 | `selectedItem`     | `any`             | the currently selected item input              |
 
+### props
+
+As a convenience, all props you pass to <Downshift /> are forwarded along to you. You can access all props via the `getStateAndHelpers` function.
```

Comment:
The `getStateAndHelpers` function is not exposed, so we shouldn't mention that. Just remove this sentence:

> You can access all props via the `getStateAndHelpers` function.

You could change this section to say:

```
### props

As a convenience, the `id` and `itemToString` props which you pass to `<Downshift />` are available here as well.
```